documentation.html

```diff
@@ -59,7 +59,194 @@
+                </div>
+                <div class="alert-text">
+                  <p>Low disk storage, clear some space.</p>
+                </div>
+              </div>
+              <!-- info-alert -->
+              <div class="alert alert-info">
+                <div class="alert-icon">
+                  <img
+                    src="./assets/alert-icons/info-alert.svg"
+                    alt="alert-success"
+                  />
+                </div>
+                <div class="alert-text">
+                  <p>An update is available to install.</p>
+                </div>
+              </div>
+              <!-- error-alert -->
+              <div class="alert alert-error">
+                <div class="alert-icon">
+                  <img
+                    src="./assets/alert-icons/error-alert.svg"
+                    alt="alert-success"
+                  />
+                </div>
+                <div class="alert-text">
+                  <p>There was an error processing your request!</p>
+                </div>
+              </div>
+            </div>
+          </div>
+
+          <div class="code-snippet">
+            <script src="https://gist.github.com/rahulk31/6b49f4a30b42990c8e711961bbf9c25c.js"></script>
+          </div>
+        </div>
+
+        <!-- Avatar Div -->
+        <div id="avatar">
+          <h2 class="primary-heading-blue">Avatars</h2>
+          <p class="para-text">
+            Avatar is basically the display picture of a user. It generally is
+            very small in size and the sizes can vary depending on the use case.
+            Stack UI provides 6 standard sizes for your avatars. Check below the
+            important classes and the way to implement.
+          </p>
+
+          <div class="component-examples">
+            <div class="avatar-wrapper">
+              <div class="avatar avatar-xs">
+                <img src="./assets/avatar/girl-with-curly-hairs.svg" alt="" />
+              </div>
+            </div>
+
+            <div class="avatar-wrapper">
+              <div class="avatar avatar-s">
+                <img src="./assets/avatar//man-smiling.svg" alt="" />
+              </div>
+            </div>
+
+            <div class="avatar-wrapper">
+              <div class="avatar avatar-m">
+                <img src="./assets/avatar/man-winking.svg" alt="" />
+              </div>
+            </div>
+
+            <div class="avatar-wrapper">
+              <div class="avatar avatar-l">
+                <img src="./assets/avatar/smiling-girl.svg" alt="" />
+              </div>
+            </div>
+
+            <div class="avatar-wrapper">
+              <div class="avatar avatar-xl">
+                <img src="./assets/avatar/marathi-woman.svg" alt="" />
+              </div>
+            </div>
+
+            <div class="avatar-wrapper">
+              <div class="avatar avatar-xxl">
+                <img src="./assets/avatar/rajasthani-male.svg" alt="" />
```

Comment:
alts are empty for images.

Reply:
Thanks Naman for pointing these, will provide some text asap 😄

Reply:
Have added alt texts:) Merging PR.